api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/EncounterTypeBundle.java

```diff
@@ -46,8 +46,7 @@ public void install() throws Exception {
         install(EncounterTypes.ANC_INTAKE);
         install(EncounterTypes.ANC_FOLLOWUP);
         install(EncounterTypes.MCH_DELIVERY);
-        install(EncounterTypes.MEXICO_CLINIC_INITIAL_VISIT);
-        install(EncounterTypes.MEXICO_CLINIC_FOLLOWUP_VISIT);
+        install(EncounterTypes.MEXICO_CLINIC_VISIT);
 
         uninstall(possible(EncounterType.class, AttachmentsConstants.ENCOUNTER_TYPE_UUID), "not used");
```

Comment:
@brandones  you'll want to add in uninstall calls like this one to clean up the old encounter types on the staging servers.